addons/account_peppol/tests/test_peppol_messages.py

```diff
@@ -412,6 +412,37 @@ def test_receive_success_peppol(self):
             move, [{
                 'peppol_move_state': 'done',
                 'move_type': 'in_invoice',
+                'journal_id': self.env.company.peppol_purchase_journal_id.id,
+                }])
+
+    def test_receive_success_peppol_multi_journal(self):
+
+        def _mocked_find_customer_email(self, _decoded_content):
+            return 'peppol@mycompany.com'
+        EdiProxyUser = type(self.env['account_edi_proxy_client.user'])
```

Comment:
> I'm using type() in a test, to patch a function. I don't see how that would be unsafe

It isn't unsafe as you are in a test and there is no way to reach that `type()` from a server action.

That being said, you can avoid it easily, and this is actually the recommended way:


```suggestion
        EdiProxyUser = self.env.registry['account_edi_proxy_client.user']
```